android/uhabits-android/src/main/java/org/isoron/uhabits/activities/habits/list/views/HabitCardListAdapter.java

```diff
@@ -83,6 +83,7 @@ public HabitCardListAdapter(@NonNull HabitCardListCache cache,
         cache.setListener(this);
         cache.setCheckmarkCount(
             ListHabitsRootViewKt.MAX_CHECKMARK_COUNT);
+        cache.setPreviousOrder(preferences.getDefaultPreviousOrder());
         cache.setOrder(preferences.getDefaultOrder());
```

Comment:
Could you please rename `order` and `previousOrder` to `primaryOrder` and `secondaryOrder` everywhere? I think this is more clear. For example, these two call would become `cache.setPrimaryOrder(...)` and `cache.setSecondaryOrder(...)`

Reply:
Done.